migrations/02-create-album-table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE Albums (
+    id SERIAL,
+    name VARCHAR(255) NOT NULL,
+    year INT NOT NULL,
+    artistId SERIAL,
+	PRIMARY KEY(id),
+	CONSTRAINT fk_artist
+		FOREIGN KEY(artistId) 
+			REFERENCES Artists(id)
+)
```

Comment:
Why is this laid out like this? Remember to use indentation to help show how deep into a function/statement your work is:
```
CREATE TABLE Albums (
  id SERIAL, 
  name VARCHAR(255) NOT NULL, 
  year INT NOT NULL, 
  artistId SERIAL, 
  PRIMARY KEY(id), 
  CONSTRAINT fk_artist FOREIGN KEY(artistId) REFERENCES Artists(id)
);
```